src/org/labkey/test/components/pipeline/PipelineTriggerWizard.java

```diff
@@ -233,12 +245,12 @@ protected ElementCache newElementCache()
 
     protected class ElementCache extends Component.ElementCache
     {
-        public ElementCache()
-        {
-            if (getWrapper().getUrlParameters().containsKey("rowId"))
-                // Wait for saved config to populate form
-                Locators.pageSignal("triggerConfigLoaded").waitForElement(getDriver(), 10000);
-        }
+//        public ElementCache()
```

Comment:
probably best to remove commented code, but it's worth considering whether or not removing this synchronizing code could cause downstream race conditions